example/main.go

```diff
@@ -60,10 +50,10 @@ func main() {
 	}
 
 	webhookHandler := githubapp.NewDefaultEventDispatcher(config.Github, prCommentHandler)
-	server.Mux().Handle(pat.Post(githubapp.DefaultWebhookRoute), webhookHandler)
 
-	// Start is blocking
-	err = server.Start()
+	http.Handle(githubapp.DefaultWebhookRoute, webhookHandler)
+	logger.Info().Msg("Starting server...")
+	err = http.ListenAndServe(fmt.Sprintf("%s:%d", config.Server.Address, config.Server.Port), nil)
```

Comment:
```suggestion

	addr := fmt.Sprintf("%s:%d", config.Server.Address, config.Server.Port)
	logger.Info().Msgf("Starting server on %s...", addr)
	err = http.ListenAndServe(addr, nil)
```

That way people don't have to refer to the configuration to know what port to use.